README.md

```diff
@@ -369,24 +369,50 @@ The `bamboo_has_role` function returns a boolean of the current|given user
 has the requested role.
 
 - `role` string
-- `user` int - User id instead of the current logged user.
+- `user` (optional) int - User id instead of the current logged user.
 
 ```twig
 {# Does the current|given user has the given role ? #}
 {{ bamboo_has_role('authenticated') ? 'TRUE' : 'FALSE' }}
 ```
 
+The `bamboo_has_roles` function returns a boolean of the current|given user
+has the requested roles.
+
+- `roles` string[]
+- `conjunction` (optional) string - The conjunction to use on each roles.
+  Only the two values 'AND' or 'OR' are allowed.
```

Comment:
Mhh I don't know. Don't you think people will use (most of the time) this feature with the current user ? And so put it at the end will make it easier to use, because they will not need to retrieve the user before asserting roles/permissions.

Reply:
I'm OK if there is there is a more strict validation on `conjunction` parameter.